frontend/pages/MDMAppleSSOCallbackPage/MDMAppleSSOCallbackPage.tsx

```diff
@@ -53,6 +57,19 @@ const EnrollmentGate = ({
         </Button>
       </div>
     );
+  } else if (initiator === "setup_experience") {
```

Comment:
Can we move this up to its own if + early return rather than throwing it as an else block here? Making that tweak means that L38 doesn't need to change at all, and we can note that the EULA shows up during SSO (assuming it does?).